src/test/codegen/sanitizer_scs_attr_check.rs

```diff
@@ -0,0 +1,12 @@
+// This tests that the shadowcallstack attribute is
+// applied when enabling the shadow-call-stack sanitizer.
+//
+// needs-sanitizer-shadow-call-stack
+// compile-flags: -Zsanitizer=shadow-call-stack
+
+#![crate_type = "lib"]
+
+// CHECK: ; Function Attrs:{{.*}}shadowcallstack
```

Comment:
This probably also wants a `CHECK-NEXT: scs` to make sure this function attr is specifically associated with this function definition.

Reply:
Good suggestion -- thanks!